mfkdf2/Cargo.toml

```diff
@@ -56,9 +56,11 @@ thiserror = { version = "2.0", default-features = false }
 serde      = { version = "1.0", default-features = false, features = ["derive"] }
 serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
 
+hex = "0.4.3"
+jsonwebtoken = { version = "10.0.0", features = [
+  "rust_crypto",
+] }
```

Comment:
can we try to do `default-features = false` everywhere? This will help us down the road so we have the absolute minimal deps if we ever want to do no-std

Reply:
nice suggestion!!